hdr_schemata/models/HDRUK/v2_2_0/Coverage.py

```diff
@@ -5,45 +5,39 @@
 from pydantic import Field
 
 
+from .annotations import annotations
+
+an = annotations.coverage
+
+
 class Coverage(BaseCoverage):
     class Config:
         extra = "forbid"
 
-    gender: Optional[List[GenderType]] = Field(
-        None, title="Gender", description="Male, Female, Other"
-    )
+    gender: Optional[List[GenderType]] = Field(None, **an.gender.__dict__)
 
     biologicalsamples: Optional[List[BiologicalSampleType]] = Field(
-        None, title="Biological Samples", description="Blood, Saliva, Urine, Other"
+        None,
+        **an.gender.__dict__,
```

Comment:
Related to comment above, should the next few of these be something other than `gender`?